<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1049</width>
    <height>895</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="label1">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>200</y>
     <width>191</width>
     <height>71</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QGraphicsView" name="graphicsView">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>10</y>
     <width>551</width>
     <height>501</height>
    </rect>
   </property>
  </widget>
  <widget class="QGraphicsView" name="graphicsView_2">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>10</y>
     <width>481</width>
     <height>501</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="button_map">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>610</y>
     <width>89</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Load Map</string>
   </property>
  </widget>
  <widget class="QPushButton" name="button_heatmap">
   <property name="geometry">
    <rect>
     <x>660</x>
     <y>600</y>
     <width>141</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Load Heat Map</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>660</x>
     <y>540</y>
     <width>151</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Generate Heat Map</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
